doc/parameters.md

```diff
@@ -186,6 +186,29 @@ before the client is throttled by adding latency.
 When set to _true_, APIcast will log the response code of the response returned by the API backend in 3scale. In some plans this information can later be consulted from the 3scale admin portal.
 Find more information about the Response Codes feature on the [3scale support site](https://access.redhat.com/documentation/en-us/red_hat_3scale/2.saas/html/analytics/response-codes-tracking).
 
+### `APICAST_SERVICES_FILTER`
```

Comment:
Should we try to find a more specific name? A name that clearly distinguishes this ENV from the one that filters by ID.

`APICAST_SERVICES_FILTER_BY_URL` ?